include/psa/crypto_types.h

```diff
@@ -188,28 +173,28 @@ typedef uint8_t psa_key_persistence_t;
 
 /** Encoding of key location indicators.
  *
- * If an implementation of this API can make calls to external
+ * If an integration of Mbed TLS can make calls to external
  * cryptoprocessors such as secure elements, the location of a key
  * indicates which secure element performs the operations on the key.
- * If an implementation offers multiple physical locations for persistent
- * storage, the location indicator reflects at which physical location
- * the key is stored.
+ * Depending on the design of the driver for the secure element, the key
+ * material may either be stored either in the secure element, or
+ * in wrapped form alongside the key metadata in the primary local storage.
  *
  * This specification defines the following values of location indicators:
  * - \c 0: primary local storage.
  *   All implementations should support this value.
  *   The primary local storage is typically the same storage area that
  *   contains the key metadata.
  * - \c 1: primary secure element.
- *   Implementations should support this value if there is a secure element
- *   attached to the operating environment.
+ *   Integrations of Mbed TLS should support this value if there is a secure
```

Comment:
Line 183: "This specification ... " should be changed as well probably as done at line 152.

Reply:
Searching for `[Tt]his specification` finds two additional occurrences (lines 135 and 336).

Reply:
There are no more occurrences of “this specification” or “implementations” in this file. I'm sure there are more places in `psa/crypto*.h` where the language should be changed now that this is solely the documentation of Mbed TLS and not the PSA specification, but fixing everything would be very much out of scope of this pull request, which is solely about lifetimes.